contracts/BulkRenewal.sol

```diff
@@ -0,0 +1,48 @@
+pragma solidity >=0.5.0;
+pragma experimental ABIEncoderV2;
+
+import "@ensdomains/ens/contracts/ENS.sol";
+import "./ETHRegistrarController.sol";
+import "@ensdomains/resolver/contracts/Resolver.sol";
+
+contract BulkRenewal {
+    bytes32 constant private ETH_NAMEHASH = 0x93cdeb708b7545dc668eb9280176169d1c33cfd8ed6f04690a0bcc88a93fc4ae;
```

Comment:
I usually like having the value of the namehash here like actually the `keccak` func call.

Reply:
@decanus In this case it's a nested series of calls - so a little cumbersome to do.

Reply:
ok fair